recipes/custom-inherit/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "custom-inherit" %}
+{% set version = "2.2.2" %}
+
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/c/{{ name }}/custom_inherit-{{ version }}.tar.gz"
+  sha256: 83c048bc3415a9e38e44e78dbe231f837aa3d4fd91b4e71443b6f6e38034f583
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

Can this be noarch? https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-builds

---

Yes, it looks like this. Would this be correct syntax?
```suggestion
  noarch: python
  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

---

yep!